### DISCUSSION TITLE
Code running time taking forever

### DISCUSSION THREAD

**Discussion by yoongschoi** [nrc-cnrc/EGSnrc#1058] [Questions]
Hi, 

I've been trying to run the sample code in the EGSnrc library using a radionuclide source: https://nrc-cnrc.github.io/EGSnrc/doc/pirs898/classEGS__RadionuclideSource.html.
I made some modifications in the geometries and it seems to be talking forever. The more I add complexity to the geometry, it is not working, but I'm still waiting because there are no error messages. this is what's happening:
![image](https://github.com/nrc-cnrc/EGSnrc/assets/113466198/24199fcb-44b8-44d3-96ac-fc9e775bb680)

Since I CAN see the geometry in egs_view, I would've encountered an error message otherwise, I want to believe it just takes long. i don't think this is normal because it never happened before with even more complicated code. Is there anything I can sort this out at this point?

Thank you.

**Comment by crcrewso** [comment]
The first thing I would do is to start with 10 histories, which should give you an error, and 100 histories which might also give you an error. 

Do you see the same behaviour then?
